homework/06LessonSix.js

```diff
@@ -76,11 +80,14 @@ function ConferenceRoom() {
 // Add a method called sayHi() that returns "Hi, I'm " + this.name + '!';
 function Person(name) {
   this.name = name;
+  this.sayHi = function() {
+    return "Hi I'm" + this.name + "!";
```

Comment:
callin this method will return:

`Hi, I'm{this.name}!`

Make sure to add a space between this.name and "I'm"